docs/tempo/website/setup/tanka.md

```diff
@@ -0,0 +1,365 @@
+
+    ```yaml
+    apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      # This name uniquely identifies the PVC. Will be used in deployment below.
+      name: minio-pv-claim
+      labels:
+        app: minio-storage-claim
+    spec:
+      # Read more about access modes here: http://kubernetes.io/docs/user-guide/persistent-volumes/#access-modes
+      accessModes:
+        - ReadWriteOnce
+      storageClassName: local-path
+      resources:
+        # This is the request for storage. Should be available in the cluster.
+        requests:
+          storage: 50Gi
+    ---
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: minio
+    spec:
+      selector:
+        matchLabels:
+          app: minio
+      strategy:
+        type: Recreate
+      template:
+        metadata:
+          labels:
+            # Label is used as selector in the service.
+            app: minio
+        spec:
+          # Refer to the PVC created earlier
+          volumes:
+            - name: storage
+              persistentVolumeClaim:
+                # Name of the PVC created earlier
+                claimName: minio-pv-claim
+          initContainers:
+            - name: create-buckets
+              image: busybox:1.28
+              command:
+                - "sh"
+                - "-c"
+                - "mkdir -p /storage/tempo-data"
+              volumeMounts:
+                - name: storage # must match the volume name, above
+                  mountPath: "/storage"
+          containers:
+            - name: minio
+              # Pulls the default Minio image from Docker Hub
+              image: minio/minio:latest
+              args:
+                - server
+                - /storage
+                - --console-address
+                - ":9001"
+              env:
+                # Minio access key and secret key
+                - name: MINIO_ACCESS_KEY
+                  value: "minio"
+                - name: MINIO_SECRET_KEY
+                  value: "minio123"
+              ports:
+                - containerPort: 9000
+                - containerPort: 9001
+              volumeMounts:
+                - name: storage # must match the volume name, above
+                  mountPath: "/storage"
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: minio
+    spec:
+      type: ClusterIP
+      ports:
+        - port: 9000
+          targetPort: 9000
+          protocol: TCP
+          name: api
+        - port: 9001
+          targetPort: 9001
+          protocol: TCP
+          name: console
+      selector:
+        app: minio
+    ```
+
+  1. Run the following command to apply the minio.yaml file:
+
+        ```bash
+        kubectl apply --namespace tempo -f minio.yaml
+        ```
+
+  1. To check that MinIO is correctly configured, sign in to MinIO and verify that a bucket has been created. Without these buckets, no data will be stored.
```

Comment:
Do these steps within steps render okay?

Reply:
Yes. It's tricky getting the alignment in Markdown to be correct, but they do render properly.